scripts/setup-caas-agents.sh

```
@@ -87,14 +133,12 @@ virt-install \
 echo "Agent VM created and booting"
 SSHEOF
 
-# Wait for agent to register
-echo "Waiting for agent to register..."
-retry_until 300 15 '[[ $(oc get agent -n ${AGENT_NAMESPACE} --no-headers 2>/dev/null | wc -l) -gt 0 ]]' || {
+echo "[6/6] Waiting for agent to register..."
+retry_until 600 10 '[[ $(oc get agent -n ${AGENT_NAMESPACE} --no-headers 2>/dev/null | wc -l) -gt 0 ]]' || {
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)